nodejs_mic_vad_streaming/Readme.md

```diff
@@ -5,40 +5,72 @@ DeepSpeech with voice activity detection.
 
 ### Prerequisites:
 
-1) The example utilized the [mic](https://github.com/ashishbajaj99/mic) NPM module which requires
-either [sox](http://sox.sourceforge.net/) (Windows/Mac) or [arecord](http://alsa-project.org/) (Linux).
+Node LTS v14.14 installed:
+```bash
+nvm install 14.14
+```
+
+#### Windows/OSX
 
-2) Download the pre-trained DeepSpeech english model (1089MB):
+1. [mic](https://github.com/ashishbajaj99/mic) NPM module - which requires [sox](http://sox.sourceforge.net/)
+1. The pre-trained DeepSpeech english model and scorer(1089MB)
 
 ```
+npm install sox mic
 wget https://github.com/mozilla/DeepSpeech/releases/download/v0.8.0/deepspeech-0.8.0-models.pbmm
 wget https://github.com/mozilla/DeepSpeech/releases/download/v0.8.0/deepspeech-0.8.0-models.scorer
 ```
 
-#### Dependency
+#### Linux
 
-Is needed the library **libasound2-dev**
+1. [arecord](http://alsa-project.org/)
+1. The library **libasound2-dev**
+1. [mic](https://github.com/ashishbajaj99/mic) NPM module - which requires [sox](http://sox.sourceforge.net/)
+1. The pre-trained DeepSpeech english model and scorer(1089MB)
 
 ```
-$ sudo apt-get install libasound2-dev
+sudo apt-get install libasound2-dev alsa-utils
+npm install mic
```

Comment:
Shoudn't that `mic` be a dependency set at the `package.json` level instead ?

Reply:
Errgh, it looks like it already is.